src/target/riscv/riscv.c

```diff
@@ -3173,6 +3253,153 @@ COMMAND_HANDLER(riscv_set_ebreaku)
 	return ERROR_OK;
 }
 
+COMMAND_HANDLER(riscv_itrigger)
+{
+	if (CMD_ARGC < 1) {
+		LOG_ERROR("Command takes at least 1 parameter");
+		return ERROR_COMMAND_SYNTAX_ERROR;
+	}
+
+	struct target *target = get_current_target(CMD_CTX);
+	const int ITRIGGER_UNIQUE_ID = -CSR_TDATA1_TYPE_ITRIGGER;
+
+	if (riscv_enumerate_triggers(target) != ERROR_OK)
+		return ERROR_FAIL;
+
+	if (!strcmp(CMD_ARGV[0], "set")) {
+		if (find_first_trigger_by_id(target, ITRIGGER_UNIQUE_ID) >= 0) {
+			LOG_ERROR("An itrigger is already set, and OpenOCD "
```

Comment:
Here it'd be good to use LOG_TARGET_ERROR:

```suggestion
			LOG_TARGET_ERROR(target, "An itrigger is already set, and OpenOCD "
```

There are more of these in `riscv_itrigger` and `riscv_etrigger`. I won't list all of them so that I don't clutter the review.

Reply:
Done. Thanks.